src/tests/ftest/datamover/dm_large_dir.yaml

```diff
@@ -21,8 +21,8 @@ server_config:
 pool:
   mode: 146
   name: daos_server
-  scm_size: 42000000000       # 42G
-  nvme_size: 700000000000     # 700G
+  scm_size: 95%
+  nvme_size: 90%
```

Comment:
@mchaarawi I have an open PR #6480 that updates both of these files to use percentages.

Reply:
i know.. i wanted to get these to pass in my test though since it's not landed.

Reply:
@mchaarawi ok I see